components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/resources/web/role/edit-users.jsp

```diff
@@ -217,10 +217,10 @@
         form.setAttribute("method", "POST");
         form.setAttribute("action", page + "?" + pageNumberParameterName + "=" + pageNumber + "&roleName=" + '<%=roleName%>');
         var selectedRolesStr = "";
-        $(["input:checkbox:checked"]).each(function(index){
+        $("input[type='checkbox']:checked").each(function(index){
             if(!$(this).is(":disabled")){
-                selectedRolesStr += $(this).val();
-                if(index != $("input:checkbox:checked").length-1){
+                selectedRolesStr += $("input[type='checkbox']:checked")[index].value;
+                if(index != $("input[type='checkbox']:checked").length-1){
                     selectedRolesStr += ":";
```

Comment:
This same block of code is found in 6 pages. Need to check all those places for consistency.

1. role/edit-users.jsp
2. role/view-users.jsp
3. role/add-step3.jsp
4. user/edit-user-roles.jsp
5. user/view-roles.jsp
6. user/add-step2.jsp